### DISCUSSION TITLE
Add a reference to the Thank-You app in the Fossify home page

### DISCUSSION THREAD

**Discussion by antodc** [FossifyOrg/General-Discussion#264] [General]
I found out only by chance that the Thank-You app was released on Google Play Store. I was unaware of it. I think that it could represent an important source of funding for this outstanding project. Perhaps a reference to that app should be added here (https://github.com/fossifyorg), where the possible alternatives to support Fossifuy are listed.

**Comment by naveensingh** [comment]
It was announced:
 - https://github.com/orgs/FossifyOrg/discussions/57#discussioncomment-10965221
 - https://www.reddit.com/r/Fossify/comments/1gllikn/fossify_camera_and_fossify_thank_you_are_out_on/

> Perhaps a reference to that app should be added here (https://github.com/fossifyorg

Thanks but I don't think it fits there, especially since it's only available on Google Play.
